synthtool/languages/node.py

```diff
@@ -177,6 +178,11 @@ def fix(hide_output=False):
     shell.run(["npm", "run", "prelint"], check=False, hide_output=hide_output)
     logger.debug("Running fix...")
     shell.run(["npm", "run", "fix"], hide_output=hide_output)
+    try:
+        shell.run(["git", "checkout", "samples"], hide_output=hide_output)
```

Comment:
Does this effectively undo any changes that the npm command made?

Reply:
@SurferJeffAtGoogle yes, it should put just that one folder  back to HEAD.

Note, that it is both npm, and linting that can touch that folder.